index.html

```diff
@@ -101,19 +199,623 @@ <h3>
               The port that the socket should be bound to. When omitted, or 0,
               any port can be used.
             </dd>
+            <div class="note">
+              The only current implementation (Chromium) ignores
+              {{TCPSocketOptions/localAddress}} and {{TCPSocketOptions/localPort}}.
```

Comment:
Don't specify it if all implementations ignore it.

Reply:
Fair! Is it better to comment it out or remove completely? I'd like to retain these fields in the chromium implementation.

Reply:
Comment them out and reference a spec issue as a TODO. Comment them out in Chromium as well so that it doesn't accept the parameters until they are actually supported.